diff --git a/intl/localename.c b/intl/localename.c
index af4c229..528ccbd 100644
--- a/intl/localename.c
+++ b/intl/localename.c
@@ -1387,7 +1387,9 @@ gl_locale_name_default (void)
 	switch (sub)
 	  {
 	  case SUBLANG_SINDHI_PAKISTAN: return "sd_PK";
+#if SUBLANG_SINDHI_AFGHANISTAN != SUBLANG_SINDHI_PAKISTAN
 	  case SUBLANG_SINDHI_AFGHANISTAN: return "sd_AF";
+#endif
 	  }
 	return "sd";
       case LANG_SINHALESE: return "si_LK";
